data/fixtures/scopes/c/disqualifyDelimiter.scope

```diff
@@ -0,0 +1,84 @@
+1 < 2
+1 > 2
+1 <= 2
+1 >= 2
+---
```

Comment:
what about pointer dereference `->`?

Reply:
you're also missing shift operators `>>`. How did you come up with these lists?

Reply:
added